visualizations/load-aware/store/parsers.py

```diff
@@ -22,14 +22,15 @@
 
 IMPORTANT_FILES = [
     "config.yaml",
-
+    "001__load__aware__scale__test/all_default_pods.json",
+    "001__load__aware__scale__test/all_trimaran_pods.json",
     "002__cluster__capture_environment/nodes.json",
     "002__cluster__capture_environment/ocp_version.yml",
     "003__cluster__dump_prometheus_db/prometheus.t*"
 ]
 
-PARSER_VERSION = "2023-05-31"
-ARTIFACTS_VERSION = "2023-05-31"
+PARSER_VERSION = "2023-07-07"
+ARTIFACTS_VERSION = "2023-07-07"
```

Comment:
don't bother with that, it's too hard to take into account 🙃 
(and it's not useful until you try to download and re-process old artifacts. That's one thing we're solving with Horreum)

Reply:
Gotcha!